frb/data/Galaxies/180916/FRB180916_host.json

```diff
@@ -20,6 +20,24 @@
         "u-r": 1.5209882369520886,
         "u-r_err": 0.060592290114077615
     },
+    "morphology": {
+        "PA": 52.63,
+        "PA_err": 28.16,
+        "b/a": 0.89,
+        "b/a_err": 0.07,
+        "dec": 65.71481316068828,
+        "dec_err": 0.20592000000000002,
+        "mag": 19.98,
```

Comment:
write explicitly the filter, e.g. r_mag: xx.

Reply:
That's one for @SunilSimha

Reply:
@keheintz, I would  not take the magnitude measurements seriously because the zero point is set to a constant (25) for all GALFIT analyses. Furthermore, the fitting procedure gets no information  about exposure time. I can take these out of the table for now.

Reply:
Ah ok! Yes, best to take them out then, so we/people don’t end up using the mags for anything (some of them seem realistic)

Reply:
removed